### PR TITLE
Fixes #2215: Turn effect_list.txt into effect_list.md

### LINKED ISSUE #2215: turn the effect list documentation into markdown
`doc/effect_list.txt` should be rewritten in markdown for easier reading in the browser.

proposed syntax:

```markdown
# Effect list

(description)


effect | description
-------|------------
0      | regular hit
1      | bash
2      | melee skill hit
3      | melee skill hit

...
```

### PR DESCRIPTION
**Pull request by akshat157**
[//]: # (**********************************)
[//]: # (** Fill in the following fields **)
[//]: # (**********************************)

[//]: # (Note: Lines beginning with syntax such as this one, are comments and will not be visible in your report!)

### Pull Request Prelude

[//]: # (Thank you for working on improving Hercules!)

[//]: # (Please complete these steps and check the following boxes by putting an `x` inside the brackets _before_ filing your Pull Request.)

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR will be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

[//]: # (Describe at length, the changes that this pull request makes.)

**Affected Branches:** 

[//]: # (Master? Slave?)
stable
**Issues addressed:**
Turn the effect list documentation into markdown

[//]: # (Issue Tracker Number if any.)
Tracker number: #2215

### Known Issues and TODO List
With respect to line 2 of Description of the effect list,
Is the `(01f3 <ID> ....... )` supposed to be `(0x1f3 <ID> ....... )` ?
If not, then this file is ready to be reviewed.

[//]: # (Insert checklist here)
[//]: # (Syntax: - [ ] Checkbox)

[//]: # (**NOTE** Enable the setting "[√] Allow edits from maintainers." when creating your pull request if you have not already enabled it.)

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

**Comment by akshat157** [comment]
You're welcome! :slightly_smiling_face:

**Comment by Emistry** [comment]
Could we add those effect constant name into the table as well?
Its odd to always refer two files just to get the right effect and the constant name since we're encourage user to user constant rather than effect id.


| Number ID | Constant Name | Description |
------------ | ------------- | -------------
| 1 | EF_NAME| Description |

**Comment by Helianthella** [comment]
@Emistry good idea, let's do it!

**Comment by Emistry** [comment]
I updated it with the constant name column.
Not sure which client work/not working for each effects, so I didnt add the 2 columns you mentioned.

Effect ID from `968+` has no description, I am suck at giving description, maybe someone else could help.

File: doc/effect_list.md [hunk 1]
```diff
@@ -8,974 +8,1104 @@ effect. You can produce these effects ingame by using the `@effect` atcommand.
...
-960     |  Small yellow explosion
-961     |  Cartoony whirlwind
-962     |  Rising fire
-963     |  Dark filter (like Stone Curse)
-964     |  Same as 920
-965     |  Small white explosion
-966     |  Bone crack
-967     |  Another little explosion
+Effect ID | Constant Name | Description
+ --: | :------------------------------| :----------------------------------
```
Comment (Helianthella) [inline doc/effect_list.md]: the table should be properly aligned:
```markdown
  ID | Constant Name                  | Description
----:|:-------------------------------|:----------------------------------
  -1 | EF_NONE                        | None
   0 | EF_HIT1                        | Regular Hit
   1 | EF_HIT2                        | Bash
```

And if you want to use `Effect ID`, the whole first column will have to be enlarged

Comment (Emistry) [reply doc/effect_list.md]: Updated.

**Pull request merged**